src/modules/RootPart.rb

```diff
@@ -1711,7 +1712,20 @@ def MountPartitions(root_device_current)
 
               Builtins.y2milestone("fstab %1", fstab)
 
-              if !(
+              if reiserfs_in_fstab?(fstab)
+                reiserfs_entries = fstab.find_all {|e| e["vfstype"].include? "reiser" }
+                mount_points = reiserfs_entries.map {|e| e["file"] }.join("\n")
+
+                message =
+                  Builtins.sformat(
+                    _("The mount points listed below are using reiserfs that is not supported anymore:\n\n%1\n\n" \
```

Comment:
The name of the filesystem is `ReiserFS`, but I am not sure if we capitalize the name in other parts of our code. Could you check it, please?